session/tidb.go

```diff
@@ -210,11 +190,17 @@ func runStmt(ctx context.Context, sctx sessionctx.Context, s sqlexec.Statement)
 		ctx = opentracing.ContextWithSpan(ctx, span1)
 	}
 	se := sctx.(*session)
+	sessVars := se.GetSessionVars()
+	var startTS uint64
```

Comment:
how about define a `type tsCollector struct { Ts []uint64 }` and pass it by `ctx` to  collect all retry ts?

Reply:
maybe we could add a new field for retry ts in slow log